modules/text/tutorials/install_tesseract/install_tesseract.markdown

```diff
@@ -0,0 +1,79 @@
+Tesseract (master) installation by Using git-bash (version>=2.14.1) and cmake (version >=3.9.1){#tutorial_install_tesseract}
+===============================================================
+
+-#  We suppose you installesd opencv and opencv_contrib using [this tutorials](http://docs.opencv.org/master/d3/d52/tutorial_windows_install.html#tutorial_windows_gitbash_build]
+
+
+-#  save this script with name installTesseract.sh in c:/lib
+@code{.bash}
+#!/bin/bash
+function MAJGitRepo
+{
+if [  ! -d "$myRepo/$1"  ]; then
+  echo "clonning ${1}"
+  git clone $2
+  mkdir Build/$1
+else
+  echo "update $1"
+  cd $1
+  git pull
+  cd ..
+fi
+}
+echo "Installation de leptonica et tesseract"
```

Comment:
echo "Installing leptonica and tesseract"

Reply:
OK